recipes/package_repo.rb

```diff
@@ -97,7 +97,11 @@
   when 'debian'
     apt_repository "mongodb-org-#{pkg_major_version}" do
       uri node['mongodb3']['package']['repo']['url']
-      distribution "#{node['lsb']['codename']}/mongodb-org/#{node['mongodb3']['package']['repo']['apt']['name']}"
+      if node['platform'] == 'ubuntu' and node['platform_version'].to_f >= 15.04
+        distribution "trusty/mongodb-org/#{node['mongodb3']['package']['repo']['apt']['name']}"
```

Comment:
http://repo.mongodb.org/apt/ubuntu/dists/xenial/mongodb-org/ exists for 16.04 so this should be `= 15.04` instead.